### DISCUSSION TITLE
UX/UI

### DISCUSSION THREAD

**Discussion by sammou00** [hyf-Group2-fp/Just4Giving#13] [General]
Everything about the UX/UI

**Comment by AnisyaPurnama** [comment]
So now I will focus on the wireframe as the basic idea of how the navigation will work, the logic of the application, etc. Will collaborate with the front-end team. I will develop the idea that already there in figma from Samir.

**Reply by sammou00** [reply]
@AnisyaPurnama  very good then ! keep Bermate and me update , we are here to help you ! regards,

**Comment by sammou00** [comment]
I think we can remove the text from the logo and modify the box to another 
![Logo](https://user-images.githubusercontent.com/67711948/116537501-9e7f0100-a8e6-11eb-9e86-cd7c6516413a.png)

**Reply by sammou00** [reply]
the link https://editor.freelogodesign.org/en/logo/edit/c028f16b766d41f28e2ea9791cf9cf8c?category=0

**Reply by AnisyaPurnama** [reply]
Is there any special meaning with the color choose?

**Reply by sammou00** [reply]
The three basic color , just more beautiful

**Comment by AnisyaPurnama** [comment]
Our Figma file link : [figma](https://www.figma.com/file/K4DX1D3uuWodbFWkBQvc28/just4giving?node-id=0%3A1)

**Reply by bermarte** [reply]
Logo first version
![Logo](https://user-images.githubusercontent.com/979362/116966457-875a5d80-acb0-11eb-801a-1165634109d6.png)

![template](https://user-images.githubusercontent.com/979362/116966760-3139ea00-acb1-11eb-986f-a15939ce66ed.png)
[logo_1.zip](https://github.com/hyf-Group2-fp/Just4Giving/files/6419273/logo_1.zip)


logo second version
![logo_0 3](https://user-images.githubusercontent.com/979362/116784213-551fe480-aa93-11eb-8828-83faaad59be9.png) and singles
![logo_0 3_single](https://user-images.githubusercontent.com/979362/116784217-5c46f280-aa93-11eb-9eb2-cdcafb646bc3.png)
![g4817-9](https://user-images.githubusercontent.com/979362/116966654-f5068980-acb0-11eb-8a04-0e9da0b9ab01.png)

[logo_0.3.svg.zip](https://github.com/hyf-Group2-fp/Just4Giving/files/6419278/logo_0.3.svg.zip)
Remember that we don't have the copyright and this is why there are watermarks, it is also not very clear to me the use they allow https://support.freelogodesign.org/hc/en-us/categories/360003253451-Copyrights

**Comment by bermarte** [comment]
color schema
![20210502_175841](https://user-images.githubusercontent.com/979362/116966395-5bd77300-acb0-11eb-9c12-0aae8e69dc24.jpg)

**Comment by bermarte** [comment]
The illustration ShaRaf made
![just4giving](https://user-images.githubusercontent.com/979362/116989578-38252480-acd2-11eb-96f9-35e4f068c715.png)

**Reply by bermarte** [reply]
to link the images just use this one: `https://cdn.jsdelivr.net/gh/hyf-Group2-fp/Just4Giving/img/file`

**Comment by bermarte** [comment]
This is another Bootstrap derived theme but this time is free https://chemaalfonso.github.io/neumorph.scss/
this is another free one https://technext.github.io/neumorphism-ui/index.html

**Comment by bermarte** [comment]
Clean-up of the Figma file (copy and paste it to your Figma file)
https://www.figma.com/file/9f4qfr0lvAgnVSwPMD1KN8/just4giving-(logic-Bermarte)?node-id=0%3A1

**Comment by sharafcs50** [comment]
VISION: Our vision is to transform the lives of unprivileged people in the society.

**Reply by AnisyaPurnama** [reply]
I will put it as the explanation in the about us

**Comment by AnisyaPurnama** [comment]
So, 

About : Just4giving is more than just a list of goods. It’s a place of one’s  dare to care, to transform the lives of unprivileged people in the society.

Vision: Every human being  have rights and deserve to have a good life.

Mision : Just4giving is a platform that welcomes underprivileged individuals or new comers 
with the aim of offering them a way to support them 
in positively material growing. 
Just4giving is also a place for good people who wants to share they stuff with others.
While doing so, providing them a place to connected is our pleasure.

**Comment by bermarte** [comment]
Svgs for Divya
[humaaans_svg.zip](https://github.com/hyf-Group2-fp/Just4Giving/files/6441445/humaaans_svg.zip)

**Comment by dirk-jacobs** [comment]
Hi, 

as an example of a site which has a crisp UI but doesn't need the stroked / shadowed fonts

https://www.framer.com/

PS: Framer is in my opinion a good alternative for Figma

**Reply by AnisyaPurnama** [reply]
Indeed! The gradient font look really nice! thank you dirk!

**Comment by AnisyaPurnama** [comment]
- Terms of use : By using the app, you consent to the terms of use, if you do not consent; do not use App site. The services provided through our JUST4GIVING App require you to open an account (sign up) . You are responsible (liable) for maintaining the confidentiality of the information you hold for your account, including your password and activity that occurs under your account; so keep your information confidential. You may be held liable for mis-use or abuse for any other user of our App due to someone else using your email, password or account as a result of your fault to keep your account information confidential. Any feedback (comment) you provided on our site shall be regarded as non-confidential, will be free to use such information uninhibitedly.

- Privacy
1. We won’t pass on your phone number to any organisation outside JUST4GIVING.
2. As a giver you can get notification in your email from the needer who interest in your item.
3. As a needer you can see the email of the giver and contact them via email.
4. If you put phone numbers,  postal addresses into your posts on JUST4GIVING, other people will be able to see them. If you're concerned about this, don't do it!

- Disclaimer
1. JUST4GIVING puts people who wants to give items in touch with people who want those items. Because of how it works, JUST4GIVING never sees or checks items and they may not be in working order. Any agreement you reach about any items will be between the person requesting items and you.
2. Please use common sense when posting to JUST4GIVING or collecting an item. Example for the safety, most collections occur at home, but if you're worried about that, arrange to meet in a cafe or other public place.
3. JUST4GIVING is a free stuff app, so you should never hand over money.
4. If you are unlucky enough to be scammed, please report it to the Police, and to us .

If someone wants to add somethig please just let me know.
